webapp/cas-server-webapp-starter-tomcat/src/test/java/org/apereo/cas/web/CasWebApplicationServletInitializerTests.java

```diff
@@ -0,0 +1,37 @@
+package org.apereo.cas.web;
+
+import org.apereo.cas.util.MockServletContext;
+
+import lombok.val;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * This is {@link CasWebApplicationServletInitializerTests}.
+ *
+ * @author Misagh Moayyed
+ * @since 6.4.0
+ */
+@Tag("WebApp")
+public class CasWebApplicationServletInitializerTests {
```

Comment:
I am not sure this is necessary. A servlet initializer only coms into play when you deploy a war to an external container. This would never be the case for the starter-tomcat module, or am I missing something?